src/sequelize/models/share.js

```diff
@@ -0,0 +1,29 @@
+export default (sequelize, DataTypes) => {
+  const Share = sequelize.define(
+    'Share',
+    {
+      userId: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+      },
+      slug: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      platform: {
+        type: DataTypes.ARRAY(DataTypes.STRING),
+        defaultValue: ['neutral']
+      },
+    }
+  );
+  Share.associate = (models) => {
+    // Association definitions here
+    Share.belongsTo(models.User, {
+      foreignKey: 'userId', as: 'author', onDelete: 'CASCADE', onUpdate: 'CASCADE'
+    });
+    Share.belongsTo(models.Article, {
+      foreignKey: 'slug', onDelete: 'CASCADE', onUpdate: 'CASCADE'
```

Comment:
Slug cannot be a foreign key because it is not a primary key in the model, share.
<img width="1101" alt="Screen Shot 2019-08-28 at 12 30 03" src="https://user-images.githubusercontent.com/34879554/63848163-97745280-c98f-11e9-926e-da20707a620f.png">

Reply:
I think a foreign key can be a PK or any unique Key in the referred model which slug is. Just run migrations afresh it will solve the issue.

Reply:
I know that you can escape the issue randomly like that but it will always be there and it might bring issues in the production. Therefore, We should get rid of it.

Reply:
@placideirandora thanks I've implemented your feedback

Reply:
Cool! So, why doesn't facebook work on sharing the article?
<img width="414" alt="Screen Shot 2019-08-28 at 15 06 13" src="https://user-images.githubusercontent.com/34879554/63858252-7dde0580-c9a5-11e9-9e40-8ed8bdcfac98.png">

Reply:
> Cool! So, why doesn't facebook work on sharing the article?
> <img alt="Screen Shot 2019-08-28 at 15 06 13" width="414" src="https://user-images.githubusercontent.com/34879554/63858252-7dde0580-c9a5-11e9-9e40-8ed8bdcfac98.png">

@placideirandora facebook requires a valid secured URL but because we are using localhost it can't work right now